app/views/password_resets/new.html.erb

```diff
@@ -1,6 +1,18 @@
-<h1>Forgot Your Password?</h1>
-<p>Enter your username and we'll send you an email containing a link you can use to reset your password.</p>
-<%= form_tag({ action: 'create' }, { class: 'form-inline' }) do -%>
-  <%= text_field_tag :username, nil, required: true, placeholder: 'Username' %>
-  <%= button_tag 'Submit', class: 'btn btn-primary' %>
-<% end -%>
+<div>
+  <h1>Forgot Your Password?</h1>
+</div>
+<div>
+  <p>Enter your username and we'll send you an email containing a link you can use to reset your password.</p>
+  <%= form_tag({ action: 'create' }, { 'data-abide' => '' }) do -%>
+    <div class="row">
+      <div class="medium-6 columns username-field">
+        <%= text_field_tag :username, nil, required: true, placeholder: 'Username' %>
```

Comment:
there needs to be a `<small class="error">Required</small>` on the line after this `text_field_tag` here to make it so it doesn't get that weird jumping behavior.

This is part of [Abide Validation](http://foundation.zurb.com/docs/components/abide.html), which I must say I'm not a huge fan of.